app/models/requirement.rb

```diff
@@ -1,10 +1,27 @@
 class Requirement < ActiveRecord::Base
+  include AASM
   belongs_to :deliverable
   validates :name, { presence: true }
   validates :deliverable, { presence: true }
 
   before_validation :set_defaults
 
+  enum status: [:created, :started, :completed]
+
+  aasm column: :status do
```

Comment:
first i started with boolean. then changed it to aasm thinking when someone picks a task/requirement, it should keep that record by calling ".start!" and in the callback , we can implement some notification logic for other members on the mission.